app/join_convo.js

```diff
@@ -1,19 +1,51 @@
 import React, { Component, } from 'react';
-import { AppRegistry, Text,View,StyleSheet,Image,TouchableHighlight,TouchableOpacity, ScrollView} from 'react-native';
-import {Actions} from 'react-native-router-flux';
+import { AppRegistry,
+    AsyncStorage, Text,View,StyleSheet,Image,TouchableHighlight,TouchableOpacity, ScrollView} from 'react-native';
 
+import PubNub from 'pubnub';
 
+const pubnub = new PubNub({
```

Comment:
I think that this should go in a general config file rather than statically programmed into this class. Because it is redundantly used. https://github.com/pubnub/react-native-tutorial/blob/master/config.js (Here is how pubnub seems to do it) #16